src/python-common/ceph/deployment/service_spec.py

```diff
@@ -1345,7 +1345,7 @@ def __init__(self,
                  client_key: Optional[str] = None,
                  client_cert: Optional[str] = None,
                  root_ca_cert: Optional[str] = None,
-                 spdk_path: Optional[str] = None,
```

Comment:
I think we previously had a discussion about removing fields from the spec file class and the risks of that. If someone had `spdk_path` set in their spec file and then upgraded to a version that removed it, cephadm will fail to load the spec causing the service and eventually the daemons to be removed. Do we know if this field was present in any released versions of ceph?